Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PowerBIEmbedded/PowerBIEmbeddedPage.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<Page
```

Comment:
Got this error:

Unhandled exception at 0x07F6DC3C (Windows.UI.Xaml.dll) in Microsoft.Toolkit.Uwp.SampleApp.exe: 0xC000027B: An application-internal exception has occurred (parameters: 0x1D2CF3B8, 0x00000006). occurred

in SharedStubs.g.cs Line 29910?

Reply:
@michael-hawker We couldn't repro this error, and it appears caused by other components?